src/Renderer/Utils/Array.h

```diff
@@ -94,6 +94,11 @@ namespace SnekVk::Utils
             data = new T[this->size];
         }
 
+        Array(int size) : size{static_cast<size_t>(size)} 
```

Comment:
Again, not sure if this implicit cast is a good idea here, but I guess we can assume in this case nobody would intentionally add a negative size? Still better to force the user to do the cast in the constructor.